### PR TITLE
Wrong document for dynamic_rnn #27981

### PR DESCRIPTION
**Pull request by shashvatshahi1998**
#27981

### CODE REVIEW

**Comment by rthadur** [comment]
@shashvatshahi1998 please resolve conflicts.

**Review by mihaimaruseac — DISMISSED** [review]

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Why did you change formatting on all of the other lines?

**Comment by shashvatshahi1998** [comment]
@mihaimaruseac  I am not getting it . I have just deleted one previous line and added 3 more lines such that their length remains less than 77.

**Comment by shashvatshahi1998** [comment]
ok @mihaimaruseac  I have removed merge conflicts and also everything is fine now.

**Review by mihaimaruseac — DISMISSED** [review]

File: tensorflow/python/util/compat.py [hunk 1]
```diff
@@ -118,6 +118,27 @@ def path_to_str(path):
 
   Returns:
     A `str` object.
+
```
Comment (mihaimaruseac) [inline tensorflow/python/util/compat.py]: You included this change here, that might cause issues on merging...

File: tensorflow/python/util/compat.py [hunk 2]
```diff
@@ -118,7 +118,6 @@ def path_to_str(path):
 
   Returns:
     A `str` object.
-
```
Comment (mihaimaruseac) [inline tensorflow/python/util/compat.py]: This shouldn't be in this PR

**Comment by shashvatshahi1998** [comment]
@mihaimaruseac , @rthadur I have removed all objectionable commits as well as merge conflicts, now you can merge this pr.

**Review by mihaimaruseac — APPROVED** [review]

**Comment by shashvatshahi1998** [comment]
@rthadur apply the tests please

**Comment by shashvatshahi1998** [comment]
@mihaimaruseac I think it will merge automatically as all remaining tests need time?

**Comment by shashvatshahi1998** [comment]
@mihaimaruseac I think it will merge automatically as all remaining tests need time?

**Comment by mihaimaruseac** [comment]
Yes, you just have to wait. Please also keep in mind that it is weekend time so not many people are working now.

**Comment by shashvatshahi1998** [comment]
Please @rthadur merge it

**Comment by shashvatshahi1998** [comment]
@mihaimaruseac why copybara check failed

File: tensorflow/python/ops/rnn.py [hunk 3]
```diff
@@ -580,7 +580,9 @@ def dynamic_rnn(cell,
         tuple of Tensors each with dimensions `[batch_size, ...]`.
     sequence_length: (optional) An int32/int64 vector sized `[batch_size]`. Used
       to copy-through state and zero-out outputs when past a batch element's
-      sequence length.  So it's more for performance than correctness.
+      sequence length.  This argument enables this API to extract
```
Comment (ebrevdo) [inline tensorflow/python/ops/rnn.py]: Change to:

This parameter enables users to extract the last valid state and properly padded outputs, so it is provided for correctness.

Comment (mihaimaruseac) [reply tensorflow/python/ops/rnn.py]: Done in ca644c9

**Pull request merged**